packages/helix-shared-config/src/schemas/sitemap-language.schema.json

```diff
@@ -26,10 +34,17 @@
       "description": "The destination resource path to store sitemap to."
     },
     "hreflang": {
-      "type": "string",
-      "title": "HREF Language",
-      "description": "The HREF language to use when listing alternates according to [RFC 5988](https://datatracker.ietf.org/doc/html/rfc5988).",
-      "pattern": "^((?:(en-GB-oed|i-ami|i-bnn|i-default|i-enochian|i-hak|i-klingon|i-lux|i-mingo|i-navajo|i-pwn|i-tao|i-tay|i-tsu|sgn-BE-FR|sgn-BE-NL|sgn-CH-DE)|(art-lojban|cel-gaulish|no-bok|no-nyn|zh-guoyu|zh-hakka|zh-min|zh-min-nan|zh-xiang))|((?:([A-Za-z]{2,3}(-(?:[A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-(?:[A-Za-z]{4}))?(-(?:[A-Za-z]{2}|[0-9]{3}))?(-(?:[A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-(?:[0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(?:x(-[A-Za-z0-9]{1,8})+))?)|(?:x(-[A-Za-z0-9]{1,8})+))$"
+      "anyOf": [
+        {
+          "$ref": "#/$defs/hreflang"
+        },
+        {
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/hreflang"
+          }
+        }
+      ]
```

Comment:
Tricky one: we need to use `anyOf` instead of the more intuitive `oneOf`, because the latter requires that *exactly* one schema is valid. But since we use `coerceTypes: 'array'` in our schema validation, a single string `s` will pass both (in pseudo-code: `s => schema.validate(s, [s]) => [1, 1]`). Thanks @tripodsan